src/patterns/start-pages/index.md.njk

```diff
@@ -0,0 +1,53 @@
+---
+title: Start pages
+description:
+section: Patterns
+theme: Pages
+aliases:
+backlog_issue_id: 111
+layout: layout-pane.njk
+---
+
+{% from "_example.njk" import example %}
+
+If you’re building a digital service, you’ll need to provide a starting point for users on GOV.UK.
+
+{{ example({group: 'patterns', item: 'start-pages', example: 'default', html: true, nunjucks: true, open: true}) }}
+
+## When to use this pattern
+
+If your service is in public beta or live, you must have a starting point hosted on GOV.UK. This will link your service to the rest of GOV.UK.
+
+## How it works
+
+Start pages include 4 main elements:
+
+1. The service name: this should help people understand what your service does and whether they need to use it - [learn more about naming your service](https://www.gov.uk/service-manual/design/naming-your-service).
+
+2. A short introduction to list things that most users will need to know, for example, what your service will let them do or how much it’ll cost.
+
+3. The ‘Start now’ call-to-action button.
+
+4. A list of other ways to access your service.
+
+This is a set pattern, so you won’t be able to customise it. 
```

Comment:
might be useful to say why? Something like 'This is a common page that lives on GOV.UK - you won't be able to customise it.'

also I think its a real shame that teams can't bring their user needs to GOV.UK to customise start pages - they would have to be very clear and well evidenced obviously.